src/System.Private.CoreLib/shared/System/Delegate.cs

```diff
@@ -11,62 +10,60 @@ namespace System
 {
     public abstract partial class Delegate : ICloneable, ISerializable
     {
-        private Delegate()
-        {
-        }
-
         public virtual object Clone() => MemberwiseClone();
 
-        public static Delegate? Combine(Delegate? a, Delegate? b)
+        public static Delegate Combine(Delegate a, Delegate b)
```

Comment:
This is removing the annotations... I think we should fix this manually and signoff the commit... @jkotas ?